offchain/rollups-events/src/broker/mod.rs

```diff
@@ -273,6 +337,11 @@ pub struct BrokerCLIConfig {
     #[arg(long, env, default_value = "redis://127.0.0.1:6379")]
     redis_endpoint: String,
 
+    /// Address list of Redis cluster nodes, defined by a single string
+    /// separated by empty spaces. If present, it supersedes `redis_endpoint`
```

Comment:
We should document that the services will discover other nodes so that the user can define a single one here, and it will work.